source/_integrations/media_player.markdown

```diff
@@ -178,3 +178,64 @@ The way media players are displayed in the frontend can be modified in the [cust
 - `tv`: Device is a television type device.
 - `speaker`: Device is speaker or stereo type device.
 - `receiver`: Device is audio video receiver type device taking audio and outputting to speakers and video to some display.
+
+## Example script
+
+The example script below combines a number of media player commands to set the output speaker for [Spotify](https://www.home-assistant.io/integrations/spotify) to 30% volume and plays a randomly shuffled track from the playlist Calming Classical.
+
+This demonstrates how steps a user would go through manually can be combined into a script that can then be called by a single button press or automation.
+
+```yaml
+# configuration.yaml
+script:
```

Comment:
I am a fan of examples, but this is an example of a script. I'm not sure if that is the correct example (we rarely use scripts actually).

I think this makes more sense / be more clear, if shown as just a sequence or automation.

Reply:
Interesting, in my mind a script is just an automation without a trigger and I'm not sure what simple trigger would suit this example.

I like that the script makes it standalone and provides tracing/debugging tools.  Even though it is a little less useful in real usage (without also adding a button in lovelace) I find it easier to test with. What do you and others use?

I really like how you can run individual script actions in the editor and I have struggled to find somewhere else you can just copy and paste a sequence. Especially without editing yaml arrays/indentation which seems to continually confuse newbies.

Sorry, lots of words. I genuinely want this to be as useful to new users as possible 😃